scitos.ais/scitos.ais.core/src/main/resources/org/hmx/scitos/ais/stylesheet.xsl

```diff
@@ -241,7 +241,7 @@ span.detail-end { float: right; }
                             <xsl:value-of select="concat(' ',@code,' ')" />
                         </xsl:for-each>
                     </xsl:variable>
-                    <xsl:value-of select="count($interview//ais:Detail[contains($childCodes,concat(' ',@code,' '))])" />
+                    <xsl:value-of select="count($interview//ais:Detail[@code and contains($childCodes,concat(' ',@code,' '))])" />
```

Comment:
Count of annotated details in parent category: sum of annotations for child categories

Fix 3: Exclude token sections surrounded by an annotated detail that don't have a specific category assigned to themselves. (checking for `code` attribute's existence)